### DISCUSSION TITLE
How to turn off invert color?

### DISCUSSION THREAD

**Discussion by Megalucent** [sumatrapdfreader/sumatrapdf#4728] [Q&A] [answered]
I open the invert color unexpectedly,but I don't konw how to close it.Could someone help me?Thank you!!!!
![屏幕截图 2024-12-20 174218](https://github.com/user-attachments/assets/7b0f93f1-c8a1-4f53-9ad5-c049936e7a9a)

**Comment by GitHubRulesOK** [comment, marked as answer]
it will be either the older `i` key or newer shifted `I` (less easy to accidentally press)

**Reply by Megalucent** [reply]
"i"key can close it.Than you!!!

**Reply by zhaoleftstop** [reply]
@GitHubRulesOK Thank you. I've been struggling for a while.
